assets/app/scripts/services/logLinks.js

```diff
@@ -50,10 +50,28 @@ angular.module('openshiftConsole')
         newTab(params);
       };
 
-      // Not currently used.
-      // new tab: path/to/current?view=textonly
-      var textOnlyLink = function() {
-        newTab({view: 'textonly'});
+      // broken up for readability:
+      var template = new URITemplate([
+        "/#/discover?",
+        "_g=(",
+          "time:(from:now-7y%2Fy,mode:relative,to:now)",
+        ")",
+        "&_a=(",
+          "columns:!(_source),",
+          "index:'{namespace}.*',",
+          "query:(",
+            "query_string:(",
+              "analyze_wildcard:!t,",
+              "query:'kubernetes_pod_name: {podname} %26%26 kubernetes_namespace_name: {namespace}'",
```

Comment:
So an example URL, once URI.js has done its magic is:
`https://kibana.example.com:1443/#/discover?_g=(time:(from:now-1w,mode:relative,to:now))&_a=(columns:!(_source),index:'container-with-two-pods.*',query:(query_string:(analyze_wildcard:!t,query:'kubernetes_pod_name:%20database-1-gymxn%20%26%26%20kubernetes_namespace_name:%20container-with-two-pods')),sort:!(time,desc))&console_back_link=https:%2F%2Flocalhost:9000%2Fproject%2Fcontainer-with-two-pods%2Fbrowse%2Fpods%2Fdatabase-1-gymxn`

I've been through Chrome, Safari, FF, IE, iOS and Android (latest) and haven't had an issue.  So far seems that URI.js is doing the right thing.